### DISCUSSION TITLE
what is the meaning of words in the char in F1-confidence and P-R

### DISCUSSION THREAD

**Discussion by aouligei** [ultralytics/yolov5#10173] [Q&A]
i always wonder what is the meaning of such word,like"all classes 0.89 at 0.000" in P_curve char ,"all classes 0.68 at 0.426" in F1_curve, i will grateful if someone help me understant this.
<img width="723" alt="image" src="https://user-images.githubusercontent.com/108796015/202175101-0591b62b-d975-45fe-bf59-8093298fdbae.png">
<img width="648" alt="image" src="https://user-images.githubusercontent.com/108796015/202175463-01e3c688-7346-4307-9434-e991effb09b0.png">

**Comment by glenn-jocher** [comment]
Blue curve is the mean of all your classes. Max F1 x and y values shown in legend.

**Reply by aouligei** [reply]
thanks for your reply!
the F1 legend above says 'all classes 0.68 at 0.426', which means that I can get the  maximum F1 value 0.68 when confidence value is 0.426,
is this a correct understanding ?

**Reply by glenn-jocher** [reply]
yes

**Comment by aouligei** [comment]
thank you very much,i should have realized it earlier.



---Original---
From: "Glenn ***@***.***&gt;
Date: Thu, Nov 17, 2022 20:41 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ultralytics/yolov5] what is the meaning of words in the char inF1-confidence and P-R (Discussion #10173)




 
yes
